app/build.gradle

```diff
@@ -107,10 +107,10 @@ android {
 dependencies {
 	implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
 
-	implementation 'androidx.appcompat:appcompat:1.1.0'
-	implementation('androidx.preference:preference:1.1.0-alpha01') {
-		because 'fixes bug with preference categories'
+	implementation('androidx.appcompat:appcompat:1.2.0-alpha03') {
+		because 'fixes bug where stopped activities would not resume'
```

Comment:
:man_facepalming:

Reply:
Risky taking an alpha version here, but I guess you don't intent to make a release soon?

Reply:
True, but otherwise DayNight theme switching would [not have worked on API level 23 and below](https://developer.android.com/jetpack/androidx/releases/appcompat#1.2.0-alpha03). The changes since 1.1.0 seem to be only bugfixes. I would not want to make a release very soon, but perhaps before 1.2.0 is stable though. After testing it heavily on different API levels, we could perhaps release as is (similar to what we did with `androidx.preference` apparently)?